src/components/Map/Map.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
 import mapboxgl from 'mapbox-gl';
+import Tooltip from '../Tooltip/Tooltip';
```

Comment:
It might be good to add an extra line here to seperate internal components from module imports. Just feels good for visual discrimination